### PR TITLE
test: update cypress tests from DS to open it in noIframe - FE-3176

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Open tests from DS folder into noIframe to get cypress tests even quicker

`accordion` (timing: was `00:13` -> `00:06` ),
`flatTable` (timing: was  `01:14` -> `00:22`),
`batchSelection` (timing was `00:11` -> `00:06`),
`drawer` (timing was   `00:38` -> `00:11` ),
`popoverContainer` (timing was   `00:27` -> `00:13`),
`search` (timing was   `00:27` -> `00:12`),
`filterableSelect` (timing was  `01:14` -> `00:11`),
`multiSelect` (timing was  `01:20` -> `00:19` ),
`simpleSelect` (timing  was `01:01` -> `00:14`),
`tileSelect` (timing was   `00:09` -> `00:05`),
`toast` (timing was  `00:38` ->`00:20` ),
### Current behaviour

Some of DS tests are opening in Iframe and take a lot of resources,  cypress tests are slower
### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: cypress/features/regression/designSystem/search.feature [hunk 1]
```diff
@@ -2,37 +2,44 @@ Feature: Design System Search component
   I want to test Design System Search component
 
   Background: Open Design System Search component page
-    Given I open Design Systems default_story "Search" component docs page
+    # Given I open design systems default_story "Search" component in no iframe
```
Comment (DlgSHi) [inline cypress/features/regression/designSystem/search.feature:5]: Do we still need this? Please remove it

File: cypress/locators/advanced-color-picker/index.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import {
   ADVANCED_COLOR_PICKER_CELL,
   SIMPLE_COLOR,
+  EXPERIMENTAL_SIMPLE_COLOR_PICKER,
```
Comment (DlgSHi) [inline cypress/locators/advanced-color-picker/index.js:4]:
```suggestion
  SIMPLE_COLOR_PICKER,
```

File: cypress/locators/advanced-color-picker/index.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 } from './locators';
 
-export const experimentalSimpleColorPickerInput = index => cy.iFrame(
+export const experimentalSimpleColorPickerInputInIframe = index => cy.iFrame(
+  EXPERIMENTAL_SIMPLE_COLOR_PICKER,
```
Comment (DlgSHi) [inline cypress/locators/advanced-color-picker/index.js:8]:
```suggestion
  SIMPLE_COLOR_PICKER,
```

File: cypress/locators/advanced-color-picker/locators.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
 // component preview locators
 export const ADVANCED_COLOR_PICKER_CELL = '[data-element="color-picker-cell"]';
 export const SIMPLE_COLOR = '[data-component="simple-color"]';
+export const EXPERIMENTAL_SIMPLE_COLOR_PICKER = '[data-component="simple-color-picker"]';
```
Comment (DlgSHi) [inline cypress/locators/advanced-color-picker/locators.js:4]:
```suggestion
export const SIMPLE_COLOR_PICKER = '[data-component="simple-color-picker"]';
```

File: cypress/locators/drawer/index.js [hunk 5]
```diff
@@ -3,6 +3,6 @@ import {
 } from './locators';
 
 // component preview locators
-export const drawerToggle = drawer => cy.iFrame(`#${drawer}`).find(DRAWER_TOGGLE);
-export const drawerSidebar = drawer => cy.iFrame(`#${drawer}`).find('div:nth-child(1)');
+export const drawerToggle = () => cy.get(DRAWER_TOGGLE);
+export const drawerSidebar = drawer => cy.get(`#${drawer}`).find('div:nth-child(1)');
```
Comment (DlgSHi) [inline cypress/locators/drawer/index.js:7]: could we move `#${drawer}` as a separate locator into `locators.js` file ?

File: cypress/locators/search/index.js [hunk 6]
```diff
@@ -1,20 +1,16 @@
...
+export const searchInnerIcon = () => searchDefault().find('button[type="button"]');
+export const searchWitchButtonInput = () => searchDefault().find('input');
+export const searchButton = () => searchDefault().find(BUTTON_DATA_COMPONENT_PREVIEW);
+export const searchInput = () => cy.get(SEARCH_COMPONENT).find('input');
+export const searchIcon = () => cy.get('button[type="button"]');
 
-// component preview locators in no iFrame
-export const searchInputNoiFrame = () => cy.get(SEARCH_COMPONENT).find('input');
+export const searchInputIframe = () => cy.iFrame(SEARCH_COMPONENT).find('input');
+export const searchIconIframe = () => cy.iFrame('button[type="button"]');
```
Comment (DlgSHi) [inline cypress/locators/search/index.js:16]: can we import already defined locator from `common` locators?

File: cypress/locators/toast/index.js [hunk 7]
```diff
@@ -1,8 +1,7 @@
 import { TOAST_PREVIEW } from './locators';
 
 // component preview locators
-export const toastPreview = () => cy.iFrame(TOAST_PREVIEW).children();
-export const toastComponentIFrame = () => cy.iFrame(TOAST_PREVIEW);
-export const toastTogglePreview = e => cy.iFrame(`#${e}`);
-
+export const toastPreview = () => cy.get(TOAST_PREVIEW).children();
+export const toastTogglePreview = e => cy.get(`#${e}`);
```
Comment (DlgSHi) [inline cypress/locators/toast/index.js:5]: Move this `#${e}` as a separate locator please :)

Comment (KatarzynaQA) [reply cypress/locators/toast/index.js:5]: It's not a permanent locator, so it's not a good idea

File: cypress/locators/drawer/index.js (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/locators/drawer/index.js:7]: It's not a permanent locator, so it's not a good idea

**Review by DlgSHi — APPROVED** [review]
Good job!

**Pull request merged**